api/routes/index.js

```diff
@@ -13,17 +13,21 @@ module.exports = function (passport) {
 		res.send('Welcome to the index page');
 	});
 
-	/* GET Registration Page */
-	router.get('/signup', function (req, res) {
-
-	});
-
 	/* POST New User */
 	router.post('/signup', function (req, res, next) {
 		passport.authenticate('signup', function (err, user, info) {
-			if (err) { return next(err); }
-			if (!user) { return res.send(info.message); }
-			return res.send(info.message);
+			if (err) { 
+				return next(err); 
+			}
+			else if (!user) {
+				return res.send(info.message);
+			} else {
+				console.log('Grabbing userDetails for: ', user.username);
```

Comment:
Instead of grabbing userDetails, would probably be more clear if you used `console.log('Creating userDetails for: ', user.username);` just show what is actually happen. I know that this is just for your own sanity checks.

Reply:
will do!